app/src/main/java/com/dessalines/thumbkey/keyboards/DEMessageEaseSymbols.kt

```diff
@@ -15,6 +29,132 @@ import com.dessalines.thumbkey.utils.KeyboardC
 import com.dessalines.thumbkey.utils.KeyboardDefinition
 import com.dessalines.thumbkey.utils.KeyboardDefinitionModes
 import com.dessalines.thumbkey.utils.SwipeDirection
+import com.dessalines.thumbkey.utils.SwipeNWay
+
+val KB_DE_MESSAGEEASE_SYMBOLS_NUMERIC_KEY_ITEM =
+    KeyItemC(
+        center =
+            KeyC(
+                display = KeyDisplay.TextDisplay("123"),
+                action = KeyAction.ToggleNumericMode(true),
+                size = FontSizeVariant.SMALL,
+                color = ColorVariant.SECONDARY,
+            ),
+        swipeType = SwipeNWay.EIGHT_WAY,
+        swipes =
+            mapOf(
+                SwipeDirection.TOP to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.ContentCopy),
+                        action = KeyAction.Copy,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.TOP_LEFT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.SelectAll),
+                        action = KeyAction.SelectAll,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.TOP_RIGHT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.ContentCut),
+                        action = KeyAction.Cut,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.LEFT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.AutoMirrored.Outlined.Undo),
+                        action = KeyAction.Undo,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.RIGHT to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.AutoMirrored.Outlined.Redo),
+                        action = KeyAction.Redo,
+                        color = ColorVariant.MUTED,
+                    ),
+                SwipeDirection.BOTTOM to
+                    KeyC(
+                        display = KeyDisplay.IconDisplay(Icons.Outlined.ContentPaste),
+                        action = KeyAction.Paste,
+                        color = ColorVariant.MUTED,
+                    ),
+            ),
+        backgroundColor = ColorVariant.SURFACE_VARIANT,
+    )
+
+val KB_DE_MESSAGEEASE_SYMBOLS_EMOJI_KEY_ITEM =
```

Comment:
Same here, you will lose all functionality later on. Use the common emoji key.

Reply:
I've tried to make it as close to MessagEase as possible.

Reply:
This isn't messageeease, we have our own custom keys.

Reply:
> This isn't messageeease, we have our own custom keys.

Indeed, you're right.

Maybe I should be even open for new keyboard-layouts - they could have advantages. Or create my own, best fitting my conditions (I'm writing with one finger, not thumb). But better to have a standard, not learning new layouts (and difficulty to forget old ones) when changing the app, even not the optimum.

Reply:
No probs! Ya if you wanted to make your own layout also, I'd be glad to add it.